core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -717,6 +735,18 @@ public static long defaultClientIdleTimeoutMillis() {
         return DEFAULT_CLIENT_IDLE_TIMEOUT_MILLIS;
     }
 
+    /**
+     * Returns the default option to whether prevent client staying in idle state when an HTTP/2 PING frame is
+     * received or not. Note that this flag has no effect if server uses HTTP/1 connection.
```

Comment:
The description for HTTP/1 is no longer valid. Could you update Javadoc here and others?

Reply:
I think it is because there is no keep-alive in HTTP/1 servers. (Not client). Can you double check?